lib/utils.pm

```diff
@@ -1196,13 +1196,9 @@ sub reconnect_s390 {
         select_console('iucvconn');
     }
     else {
-        my $r = wait_serial($login_ready, 300);
-        if ($r && $r =~ qr/Welcome to SUSE Linux Enterprise 15/) {
-            record_soft_failure('bsc#1040606');
-        }
-        elsif ($r && is_sle) {
-            $r =~ qr/Welcome to SUSE Linux Enterprise Server/ || die "Correct welcome string not found";
```

Comment:
@Soulofdestiny When removing this line but instead just looking for the login prompt further down aren't we loosing some "boot up failure investigation"? I guess we should not just make this bis step between the grub menu and the login prompt but instead look for the intermediate steps as well. WDYT?

Reply:
hm... yeah that might happen